core/controlplane/config/config.go

```diff
@@ -564,13 +567,14 @@ type Experimental struct {
 }
 
 type Admission struct {
-	PodSecurityPolicy          PodSecurityPolicy          `yaml:"podSecurityPolicy"`
-	AlwaysPullImages           AlwaysPullImages           `yaml:"alwaysPullImages"`
-	DenyEscalatingExec         DenyEscalatingExec         `yaml:"denyEscalatingExec"`
-	Initializers               Initializers               `yaml:"initializers"`
-	Priority                   Priority                   `yaml:"priority"`
-	MutatingAdmissionWebhook   MutatingAdmissionWebhook   `yaml:"mutatingAdmissionWebhook"`
-	ValidatingAdmissionWebhook ValidatingAdmissionWebhook `yaml:"validatingAdmissionWebhook"`
+	PodSecurityPolicy                    PodSecurityPolicy                    `yaml:"podSecurityPolicy"`
+	AlwaysPullImages                     AlwaysPullImages                     `yaml:"alwaysPullImages"`
+	DenyEscalatingExec                   DenyEscalatingExec                   `yaml:"denyEscalatingExec"`
+	Initializers                         Initializers                         `yaml:"initializers"`
+	Priority                             Priority                             `yaml:"priority"`
+	MutatingAdmissionWebhook             MutatingAdmissionWebhook             `yaml:"mutatingAdmissionWebhook"`
+	ValidatingAdmissionWebhook           ValidatingAdmissionWebhook           `yaml:"validatingAdmissionWebhook"`
+	OwnerReferencesPermissionEnforcement OwnerReferencesPermissionEnforcement `yaml:"OwnerReferencesPermissionEnforcement"`
```

Comment:
Nit but the yaml key should be lowerCamelCase so that it looks `ownerReferencesPermissionEnforcement` for consistency